.circleci/config.yml

```diff
@@ -651,7 +651,7 @@ jobs:
         default: large
       parallelism:
         type: integer
-        default: 8
+        default: 1
```

Comment:
This might be too extreme?  The Playwright job would take much longer with this change.

Reply:
Wanted to give this a test using 1, which basically disables paraellism. Last commit changed to 4 and it still passed 🤷🏾‍♂️